indico/modules/events/timetable/controllers/manage.py

```diff
@@ -112,3 +113,21 @@ def _process_PATCH(self):
             with track_time_changes():
                 update_timetable_entry(self.timetable_entry, updates)
         return jsonify()
+
+    def _process_DELETE(self):
+        """Delete a timetable entry"""
+        delete_timetable_entry(self.timetable_entry)
+
+
+class RHTimetableBalloon(RHTimetableREST):
+    def _process(self):
+        if not self.timetable_entry:
+            raise BadRequest('Required key missing')
+        html = None
+        if self.timetable_entry.contribution:
```

Comment:
Well, it isn't lazy-loaded, is there any chance it'll change? Anyway, I'll do it your way once again 👅